packages/cli/src/ResponseHelper.ts

```diff
@@ -190,7 +190,7 @@ export function send<T, R extends Request, S extends Response>(
 		try {
 			const data = await processFunction(req, res);
 
-			sendSuccessResponse(res, data, raw);
+			if (!res.headersSent) sendSuccessResponse(res, data, raw);
```

Comment:
Is this change necessary since saml-related controllers are not using the annotations?

Reply:
yes it's for the default auth /login endpoint to be able to redirect to the initsso endpoint